README.md

```diff
@@ -117,6 +117,22 @@ pip install staramr
 
 However, you will have to install the external dependencies (listed below) separately.
 
+## Latest Code
+
+If you wish to make use of the latest in-development version of `staramr`, you may update directly from GitHub using `pip`:
```

Comment:
That's awesome, thanks for that tip.  I was manually specifying `PYTHONPATH` to get it to load my code, didn't know about `pip install -e`.

I've added some info to the README file about this in case anyone else was interested 91f357045f5473e4af4fd897e5f0f9148700e1c1

Reply:
That's great! :)